rollup/circuitcapacitychecker/impl.go

```diff
@@ -44,9 +44,9 @@ func NewCircuitCapacityChecker(lightMode bool) *CircuitCapacityChecker {
 }
 
 // Reset resets a CircuitCapacityChecker
-func (ccc *CircuitCapacityChecker) Reset() {
-	ccc.Lock()
-	defer ccc.Unlock()
+func Reset(ccc *CircuitCapacityChecker) {
```

Comment:
Would this change the call of Reset from `ccc.Reset()` to `Reset(ccc)`?

Reply:
> Would this change the call of Reset from `ccc.Reset()` to `Reset(ccc)`?

I have not found where used the function `NewCircuitCapacityChecker`.

Reply:
I see.